spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dishes) do
+    described_class::DEFAULT_MENU.map { |dish, price| "#{dish.to_s.capitalize}: £#{'%.2f' % price}" }.join(',')
+  end
```

Comment:
I don't think this is a good test. This is the same code as in your Menu class' #view_menu method. You're testing the code against itself. I think it would be better to literally write out what you expect this code to return.